Add data upload guide onto the docs (#6)
source-cooperative/source-docs: docs/data-upload.md

# How to Upload Your Data to Source Cooperative

This guide explains how to deliver your data to Source Cooperative in a secure and simple way.  
It is written for data providers and does not require deep Amazon Web Service (AWS) knowledge.

---

## The short version (what you need to do)

You have two main ways to upload data:

1. **Upload directly in the Source Cooperative User Interface (UI)** (drag-and-drop or file selector), or  

> 2. **Use temporary AWS credentials** to upload via the AWS Command Linke Interface (CLI) or Software Development Kits (SDKs)

Reviewer: ```suggestion
2. **Use temporary AWS credentials** to upload via the AWS Command Line Interface (CLI) or Software Development Kits (SDKs)
```

For long‑term or automated access, contact the Source Cooperative team at `hello@source.coop`.

---

> ## Option 1: Upload directly in the UI (Easiest)

Reviewer: ```suggestion
## Option 1: Upload directly in the UI (easiest)
```

If you prefer not to use AWS tools, you can upload files directly in the web interface.

### How to upload via the UI

1. Go to your product page in Source Cooperative (e.g. `https://source.coop/your-org/your-product`)
2. In the top-right corner of the Product Contents card, click on the lock icon to open the dropdown menu and enable edit mode by clicking on `Edit Mode`.
3. Either:
   - Drag‑and‑drop files to the `Product Contents` card

>    - Use `Upload Files` or `Upload Directory` options in dropdown menu to upload files via operating system's file selectory

Reviewer: ```suggestion
   - Use `Upload Files` or `Upload Directory` options in dropdown menu to upload files via operating system's file selection tools
```

Your files will be uploaded automatically to your product.

This option is ideal for:

- Small or medium uploads
- One‑time deliveries
- Non‑technical users

---

## Option 2: Upload using temporary AWS credentials (recommended for larger uploads)

For larger uploads, scripting, or programmatic access, use temporary AWS credentials.

### How to get credentials

> 1. Go to your product page

Reviewer: ```suggestion
1. Go to your product page in Source Cooperative (e.g. `https://source.coop/your-org/your-product`)
```
